### ISSUE TITLE
Unable to crew upgrade

### ISSUE THREAD

**Issue by uberhacker** [skycocker/chromebrew#7079]
```
$ crew upgrade
/usr/local/lib/crew/packages/gcc.rb:35:in `<class:Gcc>': uninitialized constant Gcc::LIB_SUFFIX (NameError)

  provides 'libs', 'GCC runtime libraries', "#{LIB_SUFFIX}/*.so*"
                                               ^^^^^^^^^^
	from /usr/local/lib/crew/packages/gcc.rb:3:in `<top (required)>'
	from /usr/local/lib/crew/lib/package.rb:65:in `require_relative'
	from /usr/local/lib/crew/lib/package.rb:65:in `get_deps_list'
	from /usr/local/lib/crew/lib/package.rb:94:in `block in get_deps_list'
	from /usr/local/lib/crew/lib/package.rb:82:in `map'
	from /usr/local/lib/crew/lib/package.rb:82:in `get_deps_list'
	from /usr/local/bin/crew:1433:in `expand_dependencies'
	from /usr/local/bin/crew:717:in `block in upgrade'
	from /usr/local/bin/crew:705:in `each'
	from /usr/local/bin/crew:705:in `upgrade'
	from /usr/local/bin/crew:2040:in `upgrade_command'
	from /usr/local/bin/crew:2060:in `<main>'
```

**Comment by uberhacker** [comment]
`crew update` is also failing.
```
$ crew update
From https://github.com/skycocker/chromebrew
 * branch            master     -> FETCH_HEAD
HEAD is now at d3038e0 crew: Add multi-component support in one package (#7018)
Package lists, crew, and library updated.
Error with gcc.rb: 
uninitialized constant Gcc::LIB_SUFFIX

  provides 'libs', 'GCC runtime libraries', "#{LIB_SUFFIX}/*.so*"
                                               ^^^^^^^^^^
```

**Comment by uberhacker** [comment]
Fixed by #7080.
